Texture.from ICanvas type Issue in v7.0.0-beta.2

Was upgrading to the latest beta.2 and got this error building https://github.com/pixijs/gif

```
src/AnimatedGIF.ts:277:37 - error TS2345: Argument of type 'HTMLCanvasElement' is not assignable to parameter of type 'TextureSource | TextureSource[]'.
  Type 'HTMLCanvasElement' is missing the following properties from type 'HTMLVideoElement': disablePictureInPicture, onenterpictureinpicture, onleavepictureinpicture, playsInline, and 54 more.

277         this.texture = Texture.from(canvas, { scaleMode });
```


This now errors:
```ts
const canvas = document.createElement('canvas');
const texture = Texture.from(canvas, { scaleMode });
```

Casting works:
```ts
const canvas = document.createElement('canvas') as ICanvas;
const texture = Texture.from(canvas, { scaleMode });
```

This doesn't seem correct to me. WDYT @SuperSodaSea?

--- a/packages/settings/src/ICanvas.ts
+++ b/packages/settings/src/ICanvas.ts
@@ -125,7 +125,7 @@ export interface ICanvas extends GlobalMixins.ICanvas, Partial<EventTarget>
     dispatchEvent(event: Event): boolean;
 
     /** Parent node of the canvas. */
-    readonly parentNode?: ICanvasParentNode;
+    readonly parentNode?: ICanvasParentNode | null;
 
     /** Style of the canvas. */
     readonly style?: ICanvasStyle;

